app/controllers/home_controller.rb

```diff
@@ -1,9 +1,11 @@
 class HomeController < ApplicationController
   def index
     @posts = Post.where(status: Post.statuses[:accepted])
-                 .order(created_at: :desc)
-                 .includes({ user: { avatar_attachment: :blob } }, :tags, :cover_image_attachment)
+                 .includes({ user: { avatar_attachment: :blob } }, :tags)
     @posts = Posts::PostsFilterService.call(@posts, params)
+    @posts = Posts::PostsSortService.call(@posts, params)
+    # load cover image of first post
+    @posts.first&.cover_image if params[:page].nil? || params[:page] == '1'
```

Comment:
Because the home just display the cover image of the first post, as you can see I included `:cover_image_attachment `before in the query above it, but the bullet gem raising warning that `avoid eager loading, remove includes :cover_image_attachment in your query`, I'm not really why bullet warning that but when i remove it the bullet warning `need eager loading, include :cover_image_attachment`, but I think if I just want to display cover image of first post so I just need to load the cover image of the first post, so I put this line and it work, I dont really know why, I think maybe when we call `@posts.first&.cover_image` it will eager load the `cover_image` only for the first post.